bevy_rapier3d/examples/ray_casting3.rs

```diff
@@ -72,20 +72,23 @@ pub fn setup_physics(mut commands: Commands) {
 
 fn cast_ray(
     mut commands: Commands,
-    windows: Res<Windows>,
+    windows: Query<&Window>,
```

Comment:
Should you add `With<PrimaryWindow>` filter here to keep the previous behavior? I see you were calling `get_primary()` to get the window.

Reply:
The example already didn't support multiple windows so it's effectively the same.
I just followed what Bevy's examples do, but I'll leave it up to seb

Reply:
No worries! I'm anxious for this new release tbh. 😆

Thanks everyone!